src/test/java/bio/terra/service/snapshotbuilder/query/FieldPointerTest.java

```diff
@@ -14,7 +15,7 @@ class FieldPointerTest {
 
   @Test
   void buildVariable() {
-    TablePointer table = new TablePointer("table", null, null);
+    TablePointer table = new TablePointer("table", null, null, Function.identity());
```

Comment:
I think this is the same as
```suggestion
    TablePointer table = TablePointer.fromTableName("table");
```